client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,14 +3,48 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
-{{#if apps.length}}
-    <ul>
-        {{#each apps}}
+<div class="form-group">
+    <form class="form">
+        <label for="yourturn-search">Search:</label>
+        <div class="input-group">
+            <input
+                name="yourturn-search"
+                autofocus="autofocus"
+                value="{{term}}"
+                type="search"
+                aria-label="Enter your term"
+                data-action="search"
+                placeholder="Application" />
+            <button
+                type="submit"
+                class="btn btn-default">
+                    <i class="fa fa-search"></i> Search
+                </button>
+            </div>
+    </form>
+</div>
+<h3>Your team apps</h3>
+{{#if teamApps.length}}
+    <ul data-block="teamApps">
+        {{#each teamApps}}
+            <li>
+                <a href="/application/detail/{{this.id}}">{{this.name}}</a>
+            </li>
+        {{/each}}
+    </ul>
+{{else}}
+    No applications owned by your team.
+{{/if}}
+
+<h3>Other apps</h3>
+{{#if otherApps.length}}
+    <ul data-block="otherApps">
+        {{#each otherApps}}
```

Comment:
The other apps. Can we limit them to say, 20 initially? Would that make sense? Would it be too complicated to interact with?